README.md

```diff
@@ -205,84 +127,35 @@ Example:
 
 > Mention all of the collaborators of this project.
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Eloghene Otiede**
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FUTURE FEATURES -->
-
-## 🔭 Future Features <a name="future-features"></a>
-
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- GitHub: [@githubhandle](https://github.com/geekelo)
+- Twitter: [@twitterhandle](https://twitter.com/Geekelo_xyz)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/eloghene-otiede)
```

Comment:
- [ ] **[OPTIONAL]** Great work on following the `README` template and customizing it with your project information! I have a small suggestion to make your social media account links clearer. Instead of using **@githubhandle** for your GitHub link, you could replace it with your actual GitHub username, **@geekelo**. Similarly, you can replace the placeholders for Twitter and LinkedIn with your actual handles. This will make it easier for potential collaborators or users to find and connect with you on social media. 👌